driver-core/src/main/java/com/datastax/driver/core/ControlConnection.java

```diff
@@ -611,6 +611,17 @@ private static void updateInfo(
       InetAddress nativeAddress = row.getInet("native_address");
       int nativePort = row.getInt("native_port");
       broadcastRpcAddress = new InetSocketAddress(nativeAddress, nativePort);
+    } else if (row.getColumnDefinitions().contains("native_transport_address")) {
+      // DSE 6.8 introduced native_transport_address and native_transport_port for the
+      // listen address.  Also included is native_transport_port_ssl (in case users
+      // want to setup a different port for SSL and non-SSL conns).
+      InetAddress nativeAddress = row.getInet("native_transport_address");
+      int nativePort = row.getInt("native_transport_port");
+      if (cluster.getCluster().getConfiguration().getProtocolOptions().getSSLOptions() != null
+          && !row.isNull("native_transport_port_ssl")) {
+        nativePort = row.getInt("native_transport_port_ssl");
+      }
+      broadcastRpcAddress = new InetSocketAddress(nativeAddress, nativePort);
```

Comment:
Location of this block (and similar logic in the other blocks) was very deliberate.  We can't go underneath the base case handler of rpc_address + default port but it does seem like this specific case will be less likely than the native_address/port case above us.